packages/eslint-config-humanmade/index.js

```diff
@@ -105,6 +105,10 @@ module.exports = {
 			'singleline': 'consistent',
 		} ],
 		'react/jsx-boolean-value': [ 'error', 'never' ],
+		'react/jsx-sort-props': [ 'error', {
```

Comment:
Per discussion, let's make sure this is a warning. hm-linter treats warnings as errors for builds anyway, so this is mainly to improve the usability locally.

```suggestion
		'react/jsx-sort-props': [ 'warn', {
```